composer.json

```diff
@@ -34,7 +34,7 @@
         "propel/propel1": "~1.7",
         "symfony/yaml": "2.*",
         "symfony/translation": "~2.0",
-        "symfony/validator": "~2.0",
+        "symfony/validator": "~2.0, <2.2",
```

Comment:
In this case, BC break should be handled or support for SF <2.3 should be
dropped.

Sullivan SENECHAL
Le 13 déc. 2015 15:17, "rpiotaix" <notifications@github.com> a écrit :

> In composer.json
> <https://github.com/schmittjoh/serializer/pull/532#discussion_r47444888>:
>
> > @@ -34,7 +34,7 @@
> >          "propel/propel1": "~1.7",
> >          "symfony/yaml": "2.*",
> >          "symfony/translation": "~2.0",
> > -        "symfony/validator": "~2.0",
> > +        "symfony/validator": "~2.0, <2.2",
>
> It is related to a change in the constructor of the
> Symfony\Component\Validator\ConstraintViolation constructor, so all the
> versions starting with 2.2 are not compatible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/schmittjoh/serializer/pull/532/files#r47444888>.
>